main_test.go

```diff
@@ -3,21 +3,31 @@ package main
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"log"
 	"os"
 	"testing"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ecr"
 	"github.com/stretchr/testify/assert"
 	"github.com/upmc-enterprises/registry-creds/k8sutil"
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2"
 	coreType "k8s.io/client-go/kubernetes/typed/core/v1"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
```

Comment:
:man_facepalming:  I copy/pasta'd from some similar code that I had already written and didn't notice you had already imported that package.